.github/workflows/docker-GA-images.yml

```diff
@@ -0,0 +1,58 @@
+name: Build and Publish docker images for Hive GA
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+
+      -
+        name: 'Set up JDK 8'
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+
+      -
+        name: Hive version
+        run:  echo "HIVE_VERSION=$(mvn -f "pom.xml" -q help:evaluate -Dexpression=project.version -DforceStdout)" | awk  -F-SNAPSHOT '{print $1}'  >> $GITHUB_ENV
+
+
+      - name: Tag
+        run: echo "tag=${{ env.HIVE_VERSION }}" | awk '{print tolower($0)}' >> $GITHUB_ENV
+
+      -
+        name: Hadoop version
+        run: echo "HADOOP_VERSION=$(mvn -f "pom.xml" -q help:evaluate -Dexpression=hadoop.version -DforceStdout)" >> $GITHUB_ENV
+      -
+        name: Tez version
+        run: echo "TEZ_VERSION=$(mvn -f "pom.xml" -q help:evaluate -Dexpression=tez.version -DforceStdout)" >> $GITHUB_ENV
+
+      -
+        name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build and push Hive Image to docker hub
+        uses: docker/build-push-action@v4
+        with:
+          context: ./packaging/src/docker/
+          file: ./packaging/src/docker/Dockerfile
+          push: true
+          tags: ${{ secrets.DOCKERHUB_USER }}/hive:${{ env.tag }}
```

Comment:
cloud we add support multiple os-arch here? for example: linux/arm64 and linux/amd64

Reply:
Thanks for the review!
Sure.